packages/bolt-connection/src/bolt/stream-observers.js

```diff
@@ -303,7 +317,9 @@ class ResultStreamObserver extends StreamObserver {
         for (let i = 0; i < meta.fields.length; i++) {
           this._fieldLookup[meta.fields[i]] = i
         }
+      }
 
+      if (meta.fields != null) {
```

Comment:
```suggestion
      if (meta.fields !== undefined) {
```
Not 100% sure on this one, but wouldn't you want to strip `null` for fields as well? Sure, it's a protocol violation, so maybe it goes :boom: before this code already.

Reply:
at this point, the fields were already processed by the driver. This is only removing from the completion metadata, the one used to create the summary.